src/components/04-table/table--borderless.config.yml

```diff
@@ -0,0 +1,3 @@
+label: Borderless Table
+preview: "@uswds-framed"
+status: wip
```

Comment:
Can _probably_ remove this. I don't think these statuses are actually used.

Reply:
It's a good way to distinguish WIP components. But yeah, once they're ready to go live we can remove status.

Reply:
Removed!